skyportal/handlers/api/stream.py

```diff
@@ -178,7 +178,7 @@ def delete(self, stream_id):
 
 
 class StreamUserHandler(BaseHandler):
-    @auth_or_token
+    @permissions(["System admin"])
```

Comment:
These (and below) are not really needed anymore. These ACLs are automatically enforced by the new permissions system when the ORM issues a select query. The only difference is that the ORM would send back a 400 instead of a 403.

Reply:
That's true for these stream endpoints, yes, but doesn't hurt to have them